package.json

```diff
@@ -92,6 +89,7 @@
     "yargs": "^16.0.3"
   },
   "dependencies": {
-    "jsdom": "15.2.1"
+    "jsdom": "15.2.1",
+    "typescript": "^4.3.2"
```

Comment:
Did this make it in by accident? or is typescript a required dependency here?

Reply:
It was added automatically by the tooling; the dependencies tab on the package's page on npm doesn't indicate typescript is a dependency, but I'm not sure how I'd otherwise tell?

Reply:
Yeah I'm curious how that was added. Try removing it and re-running `npm install` and seeing if everything works fine without it.

Reply:
I tried removing it, npm install was happy, but re-running the tests added it back.

Reply:
I suspect this is related to `npm test` regenerating the TypeScript typing files in `typings/`.

I'm unclear on whether rebuilding these during test is desirable or not, but in any case if `typescript` is needed it ought to be installed in the `devDependencies` section rather than `dependencies` and—and probably permanently, so that we can be alerted to vulnerabilities, rather than being added ad-hoc by the build/test scripts.

Reply:
Yeah, moving it to devDependencies seems to do the trick - it doesn't get added back to dependencies that way. Updated accordingly!

Reply:
Oh yeah that makes sense @cpcallen.
Re-building is important as it tells you whether there is a problem with the closure to typescript conversion. I'm ok with adding that in devDependencies given that requirement.

Reply:
It would probably be good to find the code that was previously adding it, and remove it.